assets/components/LogEventSource.vue

```diff
@@ -30,6 +31,12 @@ export default {
     this.flushBuffer = debounce(this.flushNow, 250, { maxWait: 1000 });
     this.loadLogs();
   },
+  mounted() {
+    hotkeys("command+l, ctrl+l", (event, handler) => {
```

Comment:
What would happen if there are multiple panes open?

Reply:
Like mentioned, maybe adding `shift` would better so the default behavior is not cancelled.

Reply:
> What would happen if there are multiple panes open?

Good question. I just had a look and it appears the hotkey clears logs on multiple panes at once, while the button just clears logs for the pane it's on. I think we will need to keep track of the active pane when hotkey is pressed. What do you think?

Reply:
That's good actually. With keyboard shortcut I had expected for only one to work, but I guess that makes sense. I think what it does now it makes sense. Keyboard to clear all, mouse click to do the clicked one.